pkg/pillar/types/volumetypes.go

```diff
@@ -137,12 +142,19 @@ type VolumeStatus struct {
 	WWN                     string
 	Target                  zconfig.Target
 	CustomMeta              string
+	IsAppImage              bool // Does this volume contain app image
```

Comment:
```suggestion
	ContainsAppImage              bool // Does this volume contain app image
```
The comment somehow contradicts the meaning of the variable name.

Reply:
I think I will change comment than the variable.

Reply:
Done